rest_framework/routers.py

```diff
@@ -229,6 +248,7 @@ def get_urls(self):
         """
         Use the registered viewsets to generate a list of URL patterns.
         """
+        assert self._url_conf is not None, 'SimpleRouter requires Django 2.x when using path'
```

Comment:
django 2.x or higher......

Reply:
Now that I think about this, this `assert` could be removed since support for Django 1.x was dropped in master. Is it ok for you @auvipy ?